README.md

```diff
@@ -1,53 +1,122 @@
+1. Create an `index.rst` file in the new directory (so, `docs/source/mysection/index.rst`) that looks like this:
 
-To validate all Markdown files:
+    ```
+    My Section
+    ==========
 
-    ./docs/scripts/check-all-links
+    .. toctree::
+       :maxdepth: 2
+       :glob:
 
-To validate a single Markdown file:
+       *
+    ```
 
-  ./docs/scripts/markdown-check-links path/to/file.md
+1. Update the main `docs/source/index.rst` to include your new section:
 
-See the `check-all-links` scripts for an example of how to tell `markdown-check-links` about the `_static` directory.
+    ```
+    .. toctree::
+       :maxdepth: 2
 
-To check markdown syntax, we use the node-js markdown linter remark-lint. This will lint documentation markdown files according to [following syntax rules](https://github.com/remarkjs/remark-lint/blob/master/doc/rules.md)
+       networking/index.rst
+       hardware/index.rst
+       mysection/index.rst
+    ```
 
-You can install remark link on your own machine by following [these instructions](https://github.com/remarkjs/remark-lint). MOC's Remark configuration is in `.remarkrc`
+### Adding a new document
 
-## How Paths Work
+Create your document in an existing section directory. The document should start with a level 1 heading:
 
-MOC Docs are built with Sphinx using markdown parsing plugin called [recommonmark](https://recommonmark.readthedocs.io/en/latest/).
+```
+# This is my document
+```
 
-One component of recommonmark is AutoStructify which converts references in markdown files to other markdown files into url references when the markdown files are converted to html.
+## Rendering
 
-This allows the correct link to be generated when the base url is changed (eg. building docs for a different version)
+You can render the documentation locally by installing [Sphinx][] and using the `Makefile`. These instructions are appropriate for Linux and MacOS users; if you are using Windows, please see the [Windows README](README.windows.md).
 
-The url resolver can't parse extensions other than `.html`, so a link should be the relative path to the markdown file with the '.md` extention replaced with `.html`.
+[sphinx]: https://www.sphinx-doc.org/
 
-For example, to reference `contacts/MOC-Developers-Contact-Information.md` in home.md, use a markdown link to `contacts/MOC-Developers-Contact-Information.html`.
+### Requirements
 
-This will have the url `https://moc-documents.readthedocs.io/en/latest/contacts/MOC-Developers-Contact-Information.html` when the docs are built.
+In order to build the documentation you will need a recent version of Python and the `make` utility.
 
-Paths used as links must be **relative to the file where the link is**. For example to link to an image in `contacts/MOC-Developers-Contact-Information.md` the correct path would be `../_static/img/<img_name>`
+### If this is your first time rendering the documentation
+
+1. Create and activate a Python virtual environment:
```

Comment:
Not sure if this is commonly known but it might be worth mentioning what folder you should be in when you start running these commands?

Reply:
That's a reasonable suggestion; I've added a note to the docs.